### ISSUE TITLE
GetStoresUsingApiKeysInterface.php breaks swagger schema

### ISSUE THREAD

**Issue by ilnytskyi** [klevu/productsearch#50]
The schema is broken https://magento.local/rest/all/schema?services=all

```
<message>The "array" class doesn't exist and the namespace must be specified. Verify and try again.</message>
...
Magento\Webapi\Model\Config\ClassReflector->reflectClassMethods('Klevu\\Search\\Ap...', Array)
```
Because 
```
https://github.com/klevu/productsearch/blob/master/Api/Service/Account/GetStoresUsingApiKeysInterface.php
```

has return type `@return array` magento cannot build swagger schema, needs to be replaced with actual type or as workaround `@return mixed[]`

**Comment by tejasklevu** [comment]
Hi Stanislav, Thanks for highlighting this, This issue is resolved in the latest Klevu module version 2.7.4 released today. Can you please upgrade the Klevu module and check it further?

**Comment by ilnytskyi** [comment]
thanks
